Layers/Renderers/FeatureLayerCompositionRendererMultiMesh.gd

```diff
@@ -0,0 +1,71 @@
+extends LayerCompositionRenderer
+class_name FeatureLayerCompositionRendererMultiMesh
```

Comment:
Confusing naming; only the RepeatingObjectRenderer inherits from this, while the other vector layer renderers inherit from FeatureLayerCompositionRenderer.

This renderer doesn't actually have any specific MultiMesh logic in it and is more general than the FeatureLayerCompositionRenderer. Maybe we could rename this one and clarify that the FeatureLayerCompositionRenderer is the more specific (object-based) one? (Perhaps it could even inherit from this one)

Reply:
alternative idea: merge the two and decide whether or not to use multi mesh-logic based on whether we get a mesh or a scene for objects

Reply:
We will do the following:

- FeatureLayerCompositionRendererMultiMesh will be removed, the logic will be added to the RepeatingObjectRenderer so that we have a single script
- Object layers as multi meshes will be based off the VectorVegetationRenderer since that is most similar

Reply:
Clarified via [db45605](https://github.com/boku-ilen/landscapelab/pull/320/commits/db456054651f7b2f49c838ed8751fbe50971bcbd)